packages/vector_graphics_compiler/lib/src/debug_format.dart

```diff
@@ -0,0 +1,184 @@
+  final StringBuffer buffer = StringBuffer();
+
+  @override
+  void onClipPath(int pathId) {
+    buffer.writeln('DrawClip: id:$pathId');
+  }
+
+  @override
+  void onDrawImage(int imageId, double x, double y, double width, double height,
+      Float64List? transform) {
+    buffer.writeln(
+        'DrawImage: id:$imageId (Rect.fromLTWH($x, $y, $width, $height), transform: $transform)');
+  }
+
+  @override
+  void onDrawPath(int pathId, int? paintId, int? patternId) {
+    buffer.writeln('DrawPath: id:$pathId ($paintId, $patternId)');
+  }
+
+  @override
+  void onDrawText(int textId, int paintId, int? patternId) {
+    buffer.writeln('DrawText: id:$textId ($paintId, $patternId)');
+  }
+
+  @override
+  void onDrawVertices(Float32List vertices, Uint16List? indices, int? paintId) {
+    buffer.writeln('DrawVertices: $vertices ($indices, paintId: $paintId)');
+  }
+
+  @override
+  void onImage(int imageId, int format, Uint8List data) {
+    buffer.writeln(
+        'StoreImage: id:$imageId (format:$format, byteLength:${data.lengthInBytes}');
+  }
+
+  @override
+  void onLinearGradient(double fromX, double fromY, double toX, double toY,
+      Int32List colors, Float32List? offsets, int tileMode, int id) {
+    buffer.writeln(
+        'StoreGradient: id:$id Linear(from: ($fromX, $fromY), to: ($toX, $toY), '
+        'colors: [${colors.map(_intToColor).join(',')}], offsets: $offsets, tileMode: $tileMode');
+  }
+
+  @override
+  void onMask() {
+    buffer.writeln('BeginMask:');
+  }
+
+  @override
+  void onPaintObject({
+    required int color,
+    required int? strokeCap,
+    required int? strokeJoin,
+    required int blendMode,
+    required double? strokeMiterLimit,
+    required double? strokeWidth,
+    required int paintStyle,
+    required int id,
+    required int? shaderId,
+  }) {
+    // Fill
+    if (paintStyle == 0) {
+      buffer.writeln(
+          'StorePaint: id:$id Fill(${_intToColor(color)}, blendMode: $blendMode, shader: $shaderId');
+    } else {
+      buffer.writeln(
+          'StorePaint: id:$id Stroke(${_intToColor(color)}, strokeCap: $strokeCap, $strokeJoin: $strokeJoin, '
+          'blendMode: $blendMode, strokeMiterLimit: $strokeMiterLimit, strokeWidth: $strokeWidth, shader: $shaderId');
+    }
+  }
+
+  @override
+  void onPathClose() {
+    buffer.writeln('  close()');
+  }
+
+  @override
+  void onPathCubicTo(
+      double x1, double y1, double x2, double y2, double x3, double y3) {
+    buffer.writeln('  cubicTo(($x1, $y1), ($x2, $y2), ($x3, $y3)');
+  }
+
+  @override
+  void onPathFinished() {
+    buffer.writeln('EndPath:');
+  }
+
+  @override
+  void onPathLineTo(double x, double y) {
+    buffer.writeln('  lineTo($x, $y)');
+  }
+
+  @override
+  void onPathMoveTo(double x, double y) {
+    buffer.writeln('  moveTo($x, $y)');
+  }
+
+  @override
+  void onPathStart(int id, int fillType) {
+    buffer.writeln('PathStart: id:$id ${fillType == 0 ? 'Fill' : 'Stroke'}');
```

Comment:
fill type isn't the same as fill/stroke - 

```suggestion
    buffer.writeln('PathStart: id:$id ${fillType == 0 ? 'nonZero' : 'evenOdd'}');
```

Reply:
🦞

Reply:
Done